install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: provisionings.metal3.io
+spec:
+  group: metal3.io
+  names:
+    kind: Provisioning
+    listKind: ProvisioningList
+    plural: provisionings
+    singular: provisioning
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: 'Provisioning contains configuration used by the Provisioning
+        service (Ironic) to provision baremetal hosts. \n Provisioning is created
```

Comment:
nit I would just say "the metal3 provisioning services", Ironic is supposed to be an implementation detail.